11-crash-courses/02-recursion.md

```diff
@@ -0,0 +1,12 @@
+# Recursion: A Crash Course
+
+<iframe src="https://adaacademy.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=6c9b0e2a-6031-40a9-863b-af690172d08f&autoplay=false&offerviewer=true&showtitle=true&showbrand=false&captions=true&interactivity=all" height="405" width="720" style="border: 1px solid #464646;" allowfullscreen allow="autoplay"></iframe>
+
```

Comment:
Perhaps to accommodate the lack of `arr_to_bst` code in the walkthrough, we can add the code here and suggest that - provided they have screen space - they split their screen to view the video and code at the same time.

Reply:
Updated! Thank you 🙌🏾